apps/storybook/README.md

```diff
@@ -1,5 +0,0 @@
-## NameKit React storybook documentation
```

Comment:
Is it really right to fully delete the ideas here? Should they move to a readme under namekit-react and/or in other package readme files?

Reply:
This README.md was added when another approach was used to include Tailwind classes into Storybook, which is now not used anymore. Previously from this version of the README.md, no README existed in this package. 

Would you like me to rewrite this to a descriptive and guided introduction of the Storybook components library after main code development work is done?

Reply:
This README.md was added when another approach was used to include Tailwind classes into Storybook, which is now not used anymore. Previously from this version of the README.md, no README existed in this package.

Would you like me to rewrite this to a descriptive and guided introduction of the Storybook components library after main code development work is done?

Reply:
@FrancoAguzzi Ah ok. Then it's ok to just remove the readme here for now. If you think there's an important idea to communicate of course please feel welcome to include it in the readme

Reply:
@FrancoAguzzi Ah ok. Then it's ok to just remove the readme here for now. If you think there's an important idea to communicate of course please feel welcome to include it in the readme